content/blog/2020-08-10-darktable-3.2/2020-08-10-darktable-3.2.md

```diff
@@ -0,0 +1,414 @@
+
+darktable 3.2 also introduces an automatic database maintenance option (configurable in the preferences), allowing a healthier database to be maintained over time.
+
+Finally, with the various code improvements, many similar parts have been unified, allowing later a faster and easier integration of new modules and a much better consistency on similar parts (as, for example, what has been done on pickers, described above).
+Test tools have also been added, making it easier for developers to test new additions and the backward compatibility and stability of your image processing, especially when rewriting a module for better performance. 
+
+And of course, many bugs have been fixed, making this 3.2 the most stable version 3 to date. And very clearly. If you are already using a 3.0.x version, I can just recommend you to upgrade to this 3.2, if only for its superior stability.
+
+## Some new options in preferences window
+
+Apart from the new workflows presented above and the few preferences already presented, you will find some new preferences options that may be very useful. Here are the main ones:
+
+- On lighttable, you can sort film roll by number or by folder. Following the redesign of lighttable, you can also define different size categories (via size category separators) which will allow you to define a different overlay mode according to the size of the thumbnails. The size is specified in pixels and the thumbnail size is displayed in the overlay mode popover menu shown above.
+
+- In _darkroom_, it is now possible to reduce the resolution of the navigation preview, reducing CPU usage and possibly improving performance.
+**_Warning :_** as describes in the tooltip of this option, this can reduce slightly guided mask accuracy.
+
+- In miscellaneous tab, new options allow to set shortcuts if multiple instances used.
+
+- Shortcuts tab sees, finally, a search field that allows you to quickly find the shortcut you want to customize.
+
+And finally, a new lua options tab which, to date, has only one option to show/hide the lua script installer, presented below.
+
+Note that the modified preferences are saved dynamically. Closing the preferences therefore maintains the modified preferences.
+
+## Lua script manager installer
+
+Bill Ferguson, which deals with the management of Lua scripts, had made a Lua script to manage (enable/disable) Lua scripts. But until now, the script manager should be installed manually.
+
+All this is now old history. The author has just include, directly in darktable, a Lua installer. This can be enabled or disabled on first boot and, if needed, later in the preferences. The installer downloads the latest available scripts from the repository.
```

Comment:
since it's past tense `should be` should be changed to `had to be` :)

`All this is now old history` is not an idiomatic sentence in english (IMO, english is my 3rd language :wink:), the sentence should be changed to `All this is now history`. In same paragraph: `has just include` -> `had included`

Reply:
Thanks for explanation about that idiomatic sentence. I' have no doubt that your english is way better than mine. Mine is not bad but not completely fluent.